docs/integrations/role_mapping.md

```diff
@@ -0,0 +1,87 @@
+# IAM role mapping
+
+Using an EC2 instance profile enables your Hopsworks cluster to access AWS resources. This forces all Hopsworks users to
+share the instance profile role and the resource access policies attached to that role. To allow for per project access policies
+you could have your users use AWS credentials directly in their programs which is not recommended so you should instead use
+[Role chaining](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#iam-term-role-chaining).
+
+To use Role chaining, you need to first setup IAM roles in AWS:
+
+1. Create an instance profile role with policies that will allow it to assume all resource roles that we can assume from the Hopsworks
```

Comment:
This sentence is unwieldy.
Create an instance profile role that contains the different resource roles that we want to allow selected users to be able to assume in the Hopsworks cluster. In the example below, we define 4 different resource roles: `test-role`, `s3-role`, `dev-s3-role`, and `redshift` - and later we will define which users will be allowed assume which of these resources roles.